langs/arturo/build.nims

```diff
@@ -0,0 +1,633 @@
+#!/usr/bin/env nim
```

Comment:
Is this giant file really needed, where does it come from? How do we update it, etc?

Reply:
Not at all, and the plan is to get rid of it. All the obstructions are _spammed_ in your private Discord.

Reply:
Now it's no longer needed, so I'll put it where it belongs.